### PR TITLE
test: various small component test updates

### PR DESCRIPTION
**Pull request by marktnoonan**
- Closes a lot of tickets from this epic: https://cypress-io.atlassian.net/browse/UNIFY-693
- Closes https://cypress-io.atlassian.net/browse/UNIFY-893 by adding the Cy icon to the header version info.

### Additional details

These tests cover a lot of what we saw as opportunities to add component test coverage throughout the `app`, `frontend-shared` and `launchpad` packages. There are some more tickets but this covers most changes, and each individual change is pretty small.

### PR Tasks

- [x] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/app/src/settings/project/ConfigCode.cy.tsx [hunk 1]
```diff
@@ -2,24 +2,81 @@ import ConfigCode from './ConfigCode.vue'
...
+        <ConfigCode data-cy="code" config={[{
+          field: 'objectTest',
+          value: objectTest,
+          from: 'env',
+        }]} />
+      </div>))
+
+      const expectedText = `{${Object.entries(objectTest).map(([key, value]) => `'${key}': '${value}'`).join(',')},}`
+
+      cy.log(expectedText)
```
Comment (ZachJW34) [inline packages/app/src/settings/project/ConfigCode.cy.tsx:40]: Remove?

File: packages/app/src/settings/project/ConfigCode.cy.tsx [hunk 2]
```diff
@@ -2,24 +2,81 @@ import ConfigCode from './ConfigCode.vue'
...
+      const browserText = Array.isArray(browserFieldValue) ? browserFieldValue.map((b) => b.name).join(', ') : ''
+
+      cy.contains(`browsers:`).should('contain.text', browserText)
+    })
   })
 })
+
+/*
+
+*/
```
Comment (ZachJW34) [inline packages/app/src/settings/project/ConfigCode.cy.tsx:82]: Remove

File: packages/frontend-shared/src/components/InlineCodeFragment.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import InlineCodeFragment from './InlineCodeFragment.vue'
+
+describe('<InlineCodeFragment/>', () => {
+  it('renders fragment', () => {
+    cy.mount(() => <InlineCodeFragment>I am code</InlineCodeFragment>)
```
Comment (ZachJW34) [inline packages/frontend-shared/src/components/InlineCodeFragment.cy.tsx:5]: I'd wrap this component and add some padding

**Review by ZachJW34 — APPROVED** [review]
Awesome work. Once you've fixed up the failing tests, LGTM!

File: packages/app/src/settings/project/ConfigCode.cy.tsx (hunk 1 above)
Comment (marktnoonan) [reply packages/app/src/settings/project/ConfigCode.cy.tsx:40]: Sure

**Pull request merged**